tests/asserts/keywords.py

```diff
@@ -804,12 +804,6 @@ def has_reset_room(req):
                                 msg = f"🔴ERROR: Strat has exitCondition but To Node is not door or exit:{stratRef}"
                                 messages["reds"].append(msg)
                                 messages["counts"]["reds"] += 1
-                            if "leaveShinecharged" in strat["exitCondition"]:
```

Comment:
Yeah, you can look at this branch if you want to see what I had in mind for this test: https://github.com/vg-json-data/sm-json-data/compare/master...blkerby:sm-json-data:shinecharge-tests. I wanted to get the migration applied first before adding this though, because some strats need fixes. And I want to give the tests here a little more thought, to ensure good coverage of some other possible mistakes too.

Reply:
Hmm, but that test in the new branch is only checking for the shinecharge requirement. Like you point out, checking for `shineChargeFrames` is important and that could go ahead and be added now.

Reply:
Added that now, also included one for the entrance condition.